Userland/Libraries/LibGUI/TabWidget.cpp

```diff
@@ -64,6 +64,22 @@ void TabWidget::remove_widget(Widget& widget)
         on_tab_count_change(m_tabs.size());
 }
 
+void TabWidget::remove_all_tabs_other_than(Widget& widget)
```

Comment:
Let's call this `remove_all_tabs_except()`
Also please put it in a separate commit :)

Reply:
Fixed and pushed a separate commit :^)